bundles/org.openhab.binding.gpio/src/main/feature/feature.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+	Copyright (c) 2010-2020 Contributors to the openHAB project
+
+	See the NOTICE file(s) distributed with this work for additional
+	information.
+
+	This program and the accompanying materials are made available under the
+	terms of the Eclipse Public License 2.0 which is available at
+	http://www.eclipse.org/legal/epl-2.0
+
+	SPDX-License-Identifier: EPL-2.0
+
+-->
+<features name="org.openhab.binding.gpio-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-gpio" description="gpio Binding" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<feature prerequisite="true">wrap</feature>
+		<bundle dependency="true">wrap:mvn:eu.xeli/jpigpio_2.12/0.1.0</bundle>
+		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.gpio/${project.version}</bundle>
+	</feature>
```

Comment:
```suggestion
	<feature name="openhab-binding-gpio" description="gpio Binding" version="${project.version}">
		<feature>openhab-runtime-base</feature>
		<feature prerequisite="true">wrap</feature>
		<bundle dependency="true">wrap:mvn:eu.xeli/jpigpio_2.12/0.1.0</bundle>
		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.gpio/${project.version}</bundle>
	</feature>
```
```suggestion
	<feature name="openhab-binding-gpio" description="gpio Binding" version="${project.version}">
		<feature>openhab-runtime-base</feature>
		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.gpio/${project.version}</bundle>
	</feature>
```

Reply:
Are you sure that the dependency is still resolved even if I remove it from the feature.xml?

Reply:
Yes. scope `compile` embeds the dependency in the bundle.